Pcap++/header/PcapLiveDevice.h

```diff
@@ -425,59 +447,63 @@ namespace pcpp
 		 * @return True if packet was sent successfully. False will be returned in the following cases (relevant log error is printed in any case):
 		 * - Device is not opened
 		 * - Packet length is 0
-		 * - Packet length is larger than device MTU
 		 * - Packet could not be sent due to some error in libpcap/WinPcap/Npcap
+		 * This method will not check the Mtu of the packet
 		 */
 		bool sendPacket(const uint8_t* packetData, int packetDataLength);
```

Comment:
For this method it made sense in my head to have this method not parse the MTU, and to have
```c++
bool sendPacket(const uint8_t* packetData, int packetDataLength, int packetPayloadLength)
```
for parsing the MTU, in order to keep the implementations which used byte arrays simple. In hindsight, in order to have more intuitive function signatures, it is probably worth including the default parameters:
```c++
bool checkMtu = false, pcpp::LinkLayerType linkType = pcpp::pcpp::LINKTYPE_ETHERNET
```
which were mentioned earlier in the thread #598, to give the option of parsing the packets and checking MTU. I'll add that back in unless there's any reason not to.

Reply:
Yes, let's add these arguments with the default value you suggest. If `checkMtu == false` we can simply send the byte array without any check